src/wp-includes/comment.php

```diff
@@ -1852,7 +1852,12 @@ function wp_get_unapproved_comment_author_email() {
 		$comment    = get_comment( $comment_id );
 
 		if ( $comment && hash_equals( $_GET['moderation-hash'], wp_hash( $comment->comment_date_gmt ) ) ) {
-			$commenter_email = $comment->comment_author_email;
+			// The comment will only be viewable by the comment author for 1 minute.
+			$comment_preview_expires = strtotime( $comment->comment_date_gmt . '+1 minute' );
```

Comment:
We can just `$comment->comment_date_gmt + 60`because it's already a UNIX timestamp.

Reply:
`$comment->comment_date_gmt` is mysql formatted, eg `2020-05-24 04:21:27`, so `strtotime()` is required.

Reply:
Right, sorry about the noise here.

Reply:
Right, sorry about the noise here.